server/app/controllers/monitoring/MetricsController.java

```diff
@@ -40,6 +74,17 @@ public Result getMetrics() {
     var writer = new StringWriter();
 
     try {
+      ServerMetrics serverMetrics = database.getMetaInfoManager().collectMetrics();
+      serverMetrics
+          .getQueryMetrics()
+          .forEach(
+              metric -> {
+                String name = metric.getName().substring(4);
```

Comment:
**suggestion (blocking):** extract 4 to a constant and add a comment for why it's 4 and not some other value

Reply:
Done